src/pcapi/scripts/bulk_mark_incompatible_via_isbns.py

```diff
@@ -10,18 +10,30 @@
 logger = logging.getLogger(__name__)
 
 
-def process_batch(isbns: list[str]) -> None:
-    logger.info("Bulk-marking products as incompatible", extra={"isbns": isbns})
+def process_batch(isbns: list[str], is_compatible: bool) -> None:
+    logger.info("Bulk-update products isGcuCompatible", extra={"isbns": isbns, "is_compatible": is_compatible})
```

Comment:
Afin d'avoir l'info directement dans la ligne de log, je proposerai d'avoir la valeur du flag `is_compatible` dans le message:
```suggestion
    logger.info("Bulk-update products isGcuCompatible=%s", is_compatible , extra={"isbns": isbns, "is_compatible": is_compatible})
```